docker/gitlab/README.md

```diff
@@ -3,11 +3,246 @@ gitlab-docker
 
 Dockerfile source for GitLab [docker](https://docker.io) image.
 
-# Quick Start for GitLab
-The latest GitLab docker image guide can be found [here](https://gitlab.com/gitlab-org/omnibus-gitlab/-/tree/master/doc/docker).
-
 # Upstream
 This source repo is based on: https://gitlab.com/gitlab-org/omnibus-gitlab/-/tree/master/docker.
 
 # Disclaimer
+
 This is not an official Google product.
+
+# <a name="about"></a>About
+
+This image contains an installation of Gitlab.
+
+For more information, see the
+[Official Image Marketplace Page](https://console.cloud.google.com/marketplace/product/google/gitlab).
+
+### Prerequisites
+
+Configure [gcloud](https://cloud.google.com/sdk/gcloud/) as a Docker credential helper:
+
+```shell
+gcloud auth configure-docker
+```
+### Pull command
+
+```shell
+docker -- pull marketplace.gcr.io/google/gitlab:13.12
```

Comment:
Let's do with version 14. I'm publishing it now

Reply:
Fixed